src/cdk-experimental/menu/menu-bar.spec.ts

```diff
@@ -0,0 +1,175 @@
+import {ComponentFixture, TestBed, async} from '@angular/core/testing';
+import {Component} from '@angular/core';
+import {CdkMenuBar} from './menu-bar';
+import {CdkMenuModule} from './menu-module';
+import {CdkMenuItem} from './menu-item';
+import {By} from '@angular/platform-browser';
+
+describe('MenuBar', () => {
+  describe('as radio group', () => {
+    let fixture: ComponentFixture<MenuBarRadioGroup>;
+    let menuBar: CdkMenuBar;
+    let menuItems: Array<CdkMenuItem>;
+
+    beforeEach(async(() => {
+      TestBed.configureTestingModule({
+        imports: [CdkMenuModule],
+        declarations: [MenuBarRadioGroup],
+      }).compileComponents();
+
+      fixture = TestBed.createComponent(MenuBarRadioGroup);
+      fixture.detectChanges();
+
+      menuBar = fixture.debugElement.query(By.directive(CdkMenuBar)).injector.get(CdkMenuBar);
+
+      menuItems = menuBar._allItems.toArray();
+    }));
+
+    it('should toggle menuitemradio items', () => {
+      expect(menuItems[0].checked).toBeTrue();
+      expect(menuItems[1].checked).toBeFalse();
+
+      menuItems[1].trigger();
+
+      expect(menuItems[0].checked).toBeFalse();
+      expect(menuItems[1].checked).toBeTrue();
+    });
+  });
+
+  describe('as checkbox group', () => {
+    let fixture: ComponentFixture<MenuBarCheckboxGroup>;
+    let menuBar: CdkMenuBar;
+    let menuItems: Array<CdkMenuItem>;
+
+    beforeEach(async(() => {
+      TestBed.configureTestingModule({
+        imports: [CdkMenuModule],
+        declarations: [MenuBarCheckboxGroup],
+      }).compileComponents();
+
+      fixture = TestBed.createComponent(MenuBarCheckboxGroup);
+
+      menuBar = fixture.debugElement.query(By.directive(CdkMenuBar)).injector.get(CdkMenuBar);
+      fixture.detectChanges();
+
+      menuItems = menuBar._allItems.toArray();
+    }));
+
+    it('should toggle menuitemcheckbox', () => {
```

Comment:
If you remove words like 'should' from your test descriptions, you can make them more concise. (We can assume that all tests describe the intended behavior for the components under tests.)

Also, this description could be more specific. You're testing that menuitemcheckboxes are toggled _in isolation_, so if the description mentions this, you can avoid the comment further down.

Reply:
This is a convention for Angular projects; we want test description to read like a sentence ("it should...")